editoast/core_client/src/etcs_braking_curves.rs

```diff
@@ -51,6 +54,15 @@ pub struct ETCSCurves {
     pub guidance: SimpleEnvelope,
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug, ToSchema)]
+pub struct ETCSConflictCurves {
+    #[serde(flatten)]
+    #[schema(inline)]
+    pub etcs_curves: ETCSCurves,
+    #[schema(inline)]
+    pub conflict: ConflictType,
```

Comment:
As it's not a conflict per se, I'd add it:
```suggestion
    pub conflict_type: ConflictType,
```

Reply:
Done.